Library/Homebrew/cmd/test-bot.rb

```diff
@@ -427,7 +427,7 @@ def formula(formula_name)
         "#{@tap}/#{formula_name}"
       end
 
-      test "brew", "uses", "--recursive", canonical_formula_name
+      test "brew", "uses", canonical_formula_name
```

Comment:
Because it's usually unnecessary. Now if we detect missing deps, this workaround will be obsoletely.